### ISSUE TITLE
For now just a link to Lanyrd?

### ISSUE THREAD

**Issue by simonv3** [opensourcedesign/events#1]
For example: http://lanyrd.com/topics/open-source/

**Comment by garrett** [comment]
For work, we had the need to track several events and talks from multiple conferences, so I wrote an event format and some calendaring software to handle it all, as there really wasn't anything that fit our needs (and I searched all over the place before writing something).

Data & docs: https://github.com/OSAS/rh-events/

Resulting site: http://community.redhat.com/events/

It outputs events in hCal and exports all events to an .ics file that works in software like Google Calendar, Thunderbird, Evolution, and anything else that parses iCalendar files.

The current incarnation is written in Ruby and integrated into Middleman.

We use https://github.com/OSAS/middleman-springboard as a base and the community site lives in a fork at https://github.com/OSAS/community-website (which is a little bit out of date, but mainly article-wise, as we work in the codebase behind a firewall and I manually mirror the code to GitHub — and I just got back from vacation).

Basically, the same format and code should be able to easily be re-purposed. It allows many people to work on the same event "database" in flat files in a git repo. I designed it for flexibility and human-friendliness (as much as text files in git can be friendly, that is). There's a validator too, to ensure there are not syntax errors in the format.

(FWIW: I also wrote a Lanyrd importer right before going on vacation. It hasn't been published yet, but it works for everything I've tested so far.)

**Comment by lewisnyman** [comment]
Lanyrd++

**Comment by flexbox** [comment]
> The current incarnation is written in Ruby and integrated into Middleman.

:+1: This is a great news !

**Comment by simonv3** [comment]
Looks like this is for our own events :)

**Comment by jancborchardt** [comment]
Yup – we can add any open source design related events though! If you give a talk about it anywhere, for example. :)
